test/CheckedC/dump-dataflow-facts.c

```diff
@@ -0,0 +1,213 @@
+// Tests for dumping of datafow analysis for collecting facts
+//
+// RUN: %clang_cc1 -DDEBUG_DATAFLOW -fcheckedc-extension %s 2>&1 | FileCheck %s
```

Comment:
You need to add a compiler flag.    After the analysis, you'd check the compiler flag and dump the results accordingly.  You can look at the recent commit by @mgrang to see how to do this.

Reply:
Done. The new flag is -fdump_extracted_comparison_facts.